### PR TITLE
mock-pop-mining: finalize VeriBlock mining; set keystones correctly

### PR DESCRIPTION
**Pull request by eu321**

### CODE REVIEW

File: mock-pop-mining/src/main/java/org/veriblock/sdk/mock/VeriBlockBlockchain.java [hunk 1]
```diff
@@ -15,37 +15,98 @@
...
-                                                     chainHead.getTimestamp() + 1,
-                                                     // FIXME: use the difficulty calculator to set the correct difficulty
-                                                     chainHead.getDifficulty(),
-                                                     0);
-        add(newBlock);
-        blockDataStore.put(newBlock.getHash(), blockData);
-
-        return newBlock;
+            } catch (VerificationException e) {
+            }
```
Comment (Warchant) [inline mock-pop-mining/src/main/java/org/veriblock/sdk/mock/VeriBlockBlockchain.java:108]: do not ignore exceptions, at least log them

File: core/src/main/java/org/veriblock/sdk/conf/RegTestParameters.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+// https://www.veriblock.org
+// Distributed under the MIT software license, see the accompanying
+// file LICENSE or http://www.opensource.org/licenses/mit-license.php.
+
+package org.veriblock.sdk.conf;
+
+import java.math.BigInteger;
+
+public class RegTestParameters implements VeriBlockNetworkParameters {
+    private static final BigInteger MINIMUM_POW_DIFFICULTY = new BigInteger("1");
```
Comment (blackyblack) [inline core/src/main/java/org/veriblock/sdk/conf/RegTestParameters.java]: BigInteger.ONE

File: mock-pop-mining/src/main/java/org/veriblock/sdk/mock/VeriBlockBlockchain.java (hunk 1 above)
Comment (eu321) [reply mock-pop-mining/src/main/java/org/veriblock/sdk/mock/VeriBlockBlockchain.java:108]: this exception is how SerializeService signals that the block validation has failed(and thus we need to try another nonce)

**Review by Warchant — APPROVED** [review]
Fix issue with BigInteger.ONE

**Pull request merged**